Vereinsstatuten.md

```diff
@@ -0,0 +1,128 @@
+# Statuten des Vereins Rust Basel 
+
+> Version 0.1
+
+## Name und Sitz
+
+Unter dem Namen Rust Basel besteht ein Verein, der den vorliegenden Statuten und den Vorschriften in Art. 60 ff. ZGB untersteht.
+Sitz des Vereins ist Basel Stadt.
+
+## Zweck
+
+Unser Verein hat es sich zur Aufgabe gemacht, die Rust Community im Raum Basel zu fördern und eine Plattform für den Austausch unter technikaffinen Menschen zu bieten. Über diesen Verein sollen Treffen für eine Austausch organisiert und gefördert werden.
+Der Verein ist politisch und konfessionell neutral.
+Einer Umwandlung des Vereinszwecks müssen alle Vereinsmitglieder zustimmen.
+
+## Mitglieder
+
+Mitglieder des Vereins können natürliche und juristische Personen sowie Personengesellschaften sein, welche den Zweck des Vereins anerkennen und fördern.
+Der Vorstand entscheidet über die Aufnahme von Mitgliedern nach schriftlich eingereichtem Aufnahmegesuch. Der Entscheid des Vorstandes ist endgültig. Ein ablehnender Entscheid muss nicht begründet werden.
+
+## Mitgliederbeitrag
+
+Der Mitgliederbeitrag wird von der Vereinsversammlung jährlich festgelegt. Er beträgt maximal CHF______ pro Jahr.
```

Comment:
> Er beträgt maximal CHF______ pro Jahr.

Ich würde das weg lassen. Den Rest aber drin behalten. In der Vereinsversammlung wird der Beitrag vmtl. auf 0 CHF festgelegt.